README.md

```diff
@@ -12,15 +12,15 @@ The current implementation of the `istioctl upgrade` command makes a call to ext
 
 ## Making Changes
 #### How to upgrade the istio version
-In the Dockerfile, change the environmnet variable `ENV ISTIO_VERSION` to the version of istio that you would like the installer to support.
 
-In addition to making the version change in the Dockerfile, you will need to update the `istio_version_array` in `scripts/fetch_istio_releases.sh` to support upgrades in airgapped environments. To do this, you can move the previous version from the `ENV ISTIO_VERSION` in the Dockerfile to the `istio_version_array`. If it already exists there, then its likely you don't need to add a new version to the array.
+1. In the Dockerfile, change the environmnet variable `ENV ISTIO_VERSION` to the version of istio that you would like the installer to support.
+2. Add the same version of istio that you used for `ENV ISTIO_VERSION`to the `istio_version_array` in the fetch_istio_relesaes.sh script ordering from least to greatest. When doing this step, you also want to double check that the version that previously used for `ENV ISTIO_VERSION` also exists in the `istio_version_array`.  If it already exists there, then adding the new version to the array is all that you need to do. This step ensures upgrades work in airgapped environments.
 
-In general, you would want to ensure that all previous rancher-istio supported versions are present in the `istio_version_array`, in order from least to greatest. You would also want to ensure that any istio versions that were released with an out of band (OOB) rancher-istio release have been included. The easiest way to verify that all the correct versions are present is to validate the `istio_version_array` against the versions that are in the release branches of the [rancher/charts ](https://github.com/rancher/charts "rancher/charts ") or againts the existing [istio-installer image tags](https://hub.docker.com/r/rancher/istio-installer/tags "istio-installer image tags")
+	In general, you would want to ensure that all previous rancher-istio supported versions are present in the `istio_version_array`. You would also want to ensure that any istio versions that were released with an out of band (OOB) rancher-istio release have been included. The easiest way to verify that all the correct versions are present is to validate the `istio_version_aarray` against the versions that are in the release branches of the [rancher/charts ](https://github.com/rancher/charts "rancher/charts ") or againts the existing [istio-installer image tags](https://hub.docker.com/r/rancher/istio-installer/tags "istio-installer image tags")
```

Comment:
Lets simplify this whole thing to just say: the array must include all versions from the release branch